frontend/ui/lib/widgets/create_trip_widget.dart

```diff
@@ -34,7 +58,17 @@ class _CreateTripWidgetState extends State<CreateTripWidget> {
                   width: 250.0,
                   child: TextField(
```

Comment:
FYI, using a `TextFormField` here would allow you to use a validator and easily surface errors to the widget.

Reply:
I did have a problem with figuring out how to add a validator I thing I originally had a 'TextFormField' but tripped myself up when originally trying to implement a validator in the skeletons.